packages/tsconfig/react-library.json

```diff
@@ -3,8 +3,8 @@
   "display": "React Library",
   "extends": "./base.json",
   "compilerOptions": {
```

Comment:
I prefer to have these in alphabetised order. This way you can easily understand additions in the future.